README.md

```diff
@@ -84,6 +84,20 @@ export SENTRY_ISSUES_14D=False
 ```
 As with `SENTRY_AUTH_TOKEN`, all of these variables can be passed in through the `docker run -e VAR_NAME=<>` command or via the `.env` file if using Docker Compose.
 
+### Retries to get around limits
+
+The Sentry API limits the rate of requests to 3 per second, so the exporter retries on an HTTP exception.
+
+You can tweak your settings with environment variables, though default settings should work:
+
+|  Environment variable    | Value type | Default value |                         Purpose                         |
+|:------------------------:|:----------:|:-------------:|:-------------------------------------------------------:|
+| `SENTRY_RETRY_TRIES`     | Integer    | 3             | How many retries should be made in case of an exception |
+| `SENTRY_RETRY_DELAY`     | Float      | 1             | How many seconds to wait between retries                |
+| `SENTRY_RETRY_MAX_DELAY` | Float      | 10            | Max delay to wait between retries                       |
+| `SENTRY_RETRY_BACKOFF`   | Float      | 2             | Multiplier applied to delay between attempts            |
+| `SENTRY_RETRY_JITTER`    | Float      | 0.5           | Extra seconds added to delay between attempts           |
+
```

Comment:
Could you please move this section to start at line `107` since this is a workaround to handle a known limitation on Sentry API?

Reply:
I wasn't sure if you meant line 107 as it was before or with the block, but it's been moved to the `## ℹ️ Important Notes` section, after `### Limitations`, if that's not right let me know and I'll change it :)

Reply:
It's fine where you put, thanks!